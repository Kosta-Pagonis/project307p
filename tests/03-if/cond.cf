int a;
<<<<<<< HEAD
=======
int b;
>>>>>>> d68aeae57ff6fd3d10adab4bd85b031b409a0028

a = 5;

if (a < 3) {
  a = a-1;
} else {
  a = a+1;
}

print_int(a);
<<<<<<< HEAD

=======
print_str("\n");

if (!(a < 2) && a == 6)
  b = a + 2;

print_int(b);
>>>>>>> d68aeae57ff6fd3d10adab4bd85b031b409a0028
