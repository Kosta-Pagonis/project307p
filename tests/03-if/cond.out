6
<<<<<<< HEAD
=======
8
>>>>>>> d68aeae57ff6fd3d10adab4bd85b031b409a0028
